hack/test-integration.sh

```diff
@@ -20,8 +20,6 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
```

Comment:
I found that KUBE_ROOT was not used in the script, so I removed it.
ref: https://github.com/koalaman/shellcheck/wiki/SC2034